### PR TITLE
[plugin.audio.ramfm] v2.1.0

### PR DESCRIPTION
**Pull request by spoyser**
### Description
v2.1.0:
  - Updated due to changes on RAM FM site   
  - Fixed podcasts due to website changes
  - Added back artist artwork
### Checklist:

- [ x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout gotham
git reset --hard upstream/gotham
git checkout origin/gotham -- plugin.audio.ramfm/
git commit -m '[plugin.audio.ramfm] 2.1.0'
git push -f origin gotham
```

**Comment by Rechi** [comment]
You are sure, that upstream remote points to the xbmc repo?
Post the output of `git remote -v` if you aren't sure.

**Comment by spoyser** [comment]
> You are sure, that upstream remote points to the xbmc repo?
> Post the output of `git remote -v` if you aren't sure.

I was obviously doing something wrong.
Went through it again and seems to have worked this time.
Thanks.

**Comment by Rechi** [comment]
Please migrate strings.xml to the strings.po format. You can find more info at https://kodi.wiki/view/Language_support.

File: plugin.audio.ramfm/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# Kodi Media Center language file
```
[the remark this answers is not in the document]
Comment (spoyser) [reply plugin.audio.ramfm/resources/language/resource.language.en_gb/strings.po]: Hopefully all sorted  now :)

File: plugin.audio.ramfm/default.py [hunk 2]
```diff
@@ -204,22 +205,24 @@ def ShowPodcasts():
...
+    fanart = image
 
     u   = sys.argv[0]
     u  += '?url='  + urllib.quote_plus(link)
     u  += '&mode=' + str(_PLAYPODCAST)
     u  += '&name=' + urllib.quote_plus(name)
-    liz = xbmcgui.ListItem(name, iconImage=thumbnail, thumbnailImage=thumbnail)
+    liz = xbmcgui.ListItem(name, iconImage=image, thumbnailImage=image)
+
+    liz.setProperty('Fanart_Image', fanart)
```
Comment (Rechi) [inline plugin.audio.ramfm/default.py]: better use setArt instead

File: plugin.audio.ramfm/default.py [hunk 3]
```diff
@@ -430,11 +459,19 @@ def addAvailable(title, artist, image, request):
     u   += '&mode=' + str(_TRACK)        
     liz  = xbmcgui.ListItem(name, iconImage=image, thumbnailImage=image)
 
-    xbmcplugin.addDirectoryItem(handle = int(sys.argv[1]), url = u, listitem = liz, isFolder = False)
+    liz.setProperty('Fanart_Image', fanart)
```
Comment (Rechi) [inline plugin.audio.ramfm/default.py]: same

File: plugin.audio.ramfm/default.py [hunk 4]
```diff
@@ -445,7 +482,9 @@ def addUnavailable(title, artist, image, reason):
     u   += '?mode=' + str(mode)
     liz  = xbmcgui.ListItem(name, iconImage=image, thumbnailImage=image)
 
-    xbmcplugin.addDirectoryItem(handle = int(sys.argv[1]), url = u, listitem = liz, isFolder = False) 
+    liz.setProperty('Fanart_Image', fanart)
```
Comment (Rechi) [inline plugin.audio.ramfm/default.py]: same

File: plugin.audio.ramfm/default.py [hunk 5]
```diff
@@ -523,6 +562,8 @@ def addLetter(letter):
     u        += '&mode='   + str(_LETTER)
     liz       = xbmcgui.ListItem(letter, iconImage=thumbnail, thumbnailImage=thumbnail)
 
+    liz.setProperty('Fanart_Image', FANART)
```
Comment (Rechi) [inline plugin.audio.ramfm/default.py]: same

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Please squash your commits to a single one, with the correct commit message. Then this update is ready to get accepted.

**Comment by Rechi** [comment]
@spoyser you messed something up. The easiest way to get a single commit is to execute the commands I gave above again.

**Comment by Rechi** [comment]
Sorry for the faulty command.
```
git rm -- plugin.audio.ramfm/resources/language/English/strings.xml
git commit --amend
git push -f origin gotham
```

**Pull request merged**